specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/azureWebCategory.json

```diff
@@ -0,0 +1,184 @@
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the azureWebCategory."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Expands resourceIds back referenced by the azureWebCategory resource."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting azureWebCategory resource.",
+            "schema": {
+              "$ref": "#/definitions/AzureWebCategory"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get_azureWebCategory": {
+            "$ref": "./examples/azureWebCategoryGet.json"
+          }
+        }
+     }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/azureWebCategories": {
+      "get": {
+        "tags": [
+          "AzureWebCategories"
+        ],
+        "operationId": "AzureWebCategories_ListAll",
+        "description": "Gets all the Azure Web Categorys in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of Azure Web Category resources.",
+            "schema": {
+              "$ref": "#/definitions/AzureWebCategoryListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all Azure Web Categorys for a given subscription": {
+            "$ref": "./examples/AzureWebCategoriesListBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureWebCategoryPropertiesFormat": {
+      "properties": {
+        "group": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the group that the category belongs to."
+        }
+      },
+      "description": "Azure Web Category Properties."
+    },
+    "AzureWebCategory": {
```

Comment:
Since this is a resource type, it should use another model (named Resource) to model the generic resource enevelope, and inherit that schema using allOf - here is an example: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/virtualNetwork.json#L1774

Reply:
This is a global resource where we do not need location to be specified. Pointing you to this commit -https://github.com/Azure/azure-rest-api-specs/pull/10918/commits/b5e1f0980a07af3eb0e386bbfc41a69110856d6c where I moved away from "Resource" based on reviewers feedback.

Reply:
The Approved PR to master is here: https://github.com/Azure/azure-rest-api-specs/pull/10918 (BTW, please help merge this as well)